apps/users/models.py

```diff
@@ -145,6 +152,18 @@ def create_superuser(self, username, email, password):
         GroupUser.objects.create(user=user, group=admins)
         return user
 
+    def _generate_username(self, seed):
+        log.info('Generating username for {}'.format(seed))
+        if self.model.objects.filter(username=seed).exists():
+            username = '{seed}-{num}'.format(
+                seed=seed, num=random.randint(1000, 9999))
+            log.warning('Username taken for {} trying {}'.format(
+                seed, username))
+            return username
```

Comment:
This looks dangerous:
- why this range?
- are we safe regarding the max field size?
- what if the username with this seed exist, will it simply blow up?
- why do we need this, do we have usernames that are emails from other users?
If we do need that, should we instead do something "exhaustive" and predictable like https://github.com/mozilla/olympia/blob/master/apps/addons/models.py#L98 ?